### PR TITLE
Separate r from net

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: R/rClr-exported.R [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #' @return nothing is returned by this function
 #' @export
 clrShutdown <- function() { # TODO: is this even possible given runtime's constraints?
-  result <- .C("rclr_shutdown_clr", PACKAGE=nativePkgName)
+  result <- .C("rclr_shutdown_clr", PACKAGE = nativePkgName)
```
Comment (PavelBal) [inline R/rClr-exported.R:8]: All changes here are because of styler

File: R/rClr-internal.R [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Full type name of the reflection helper the interop code written in C#
```
Comment (PavelBal) [inline R/rClr-internal.R:1]: Styler changes in all R files

**Comment by PavelBal** [comment]
This PR is about removing the C source code and all associated files.

**Comment by PavelBal** [comment]
The PR is based on the `develop` branch.

**Comment by msevestre** [comment]
👀

**Comment by msevestre** [comment]
I am so happy to see some action in this REPO I am going to have a little dance

File: R/zzz.R [hunk 3]
```diff
@@ -1,126 +1,112 @@
...
-      if( Sys.which(msvcrFileName) == '') {
+  dlls <- list.files(archLibPath, pattern = ext)
+  if (Sys.info()[["sysname"]] == "Windows") {
+      msvcrFileName <- "msvcp140.dll"
+      if (Sys.which(msvcrFileName) == "") {
         stop(paste(msvcrFileName, "was not found on this Windows system.",
           "You are probably missing the Visual C++ Redistributable for Visual Studio 2019.",
-          "Go to https://visualstudio.microsoft.com/downloads/ and dowload 'Microsoft Visual C++ Redistributable for Visual Studio 2019'", 
-          sep="\n"))
+          "Go to https://visualstudio.microsoft.com/downloads/ and dowload 'Microsoft Visual C++ Redistributable for Visual Studio 2019'",
```
Comment (Yuri05) [inline R/zzz.R:52]: This link is better: https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist

File: tests/testthat/datetime-functions.r [hunk 4]
```diff
@@ -2,67 +2,75 @@
...
+    ifelse(clrGetNativeLibName() == "rClrMono",
       # As of Mono 3.8.0, and probably earlier releases including 3.4.0, the time zone names have changed. Not Olson DB anymore. Not MS.NET either. *Sigh*
-      'E. Australia Standard Time', 
+      "E. Australia Standard Time",
       # I think even on Linux Mono does not use the Olson DB names. If still, use something like the following line
-      #  'E. Australia Standard Time', tzIdR_AUest), 
-      "AUS Eastern Standard Time") # TODO: is 'Australia/Sydney' also OK for MS.NET?
+      #  'E. Australia Standard Time', tzIdR_AUest),
+      "AUS Eastern Standard Time"
+    ) # TODO: is 'Australia/Sydney' also OK for MS.NET?
```
Comment (Yuri05) [inline tests/testthat/datetime-functions.r:29]: omg

File: tests/testthat/test-basic.r [hunk 5]
```diff
@@ -93,110 +99,105 @@ test_that("Basic types of length zero are marshalled correctly", {
...
-  expect_true(callTestCase( "ComplexEquals", z, 1, 2 ))
-  z = c(1+2i, 3+4i, 3.3+4.4i)
-  expect_equal(callTestCase( "CreateComplex", c(1,3,3.3), c(2,4,4.4)), z)
-  expect_true(callTestCase( "ComplexEquals", z, c(1,3,3.3), c(2,4,4.4)))
-  
-
+  z <- 1 + 2i
+  expect_equal(callTestCase("CreateComplex", 1, 2), z)
+  expect_true(callTestCase("ComplexEquals", z, 1, 2))
+  z <- c(1 + 2i, 3 + 4i, 3.3 + 4.4i)
```
Comment (Yuri05) [inline tests/testthat/test-basic.r:198]: I miss the tests for Quaternions 🤣

**Review by Yuri05 — APPROVED** [review]
would be good to fix the VC++Redist download link.
Apart from this, didn't see anything suspicious (not that I would understand it in detail or be sure that it still works after that 😆 )

File: R/zzz.R (hunk 3 above)
Comment (PavelBal) [reply R/zzz.R:52]: Will update in next PR

File: tests/testthat/datetime-functions.r (hunk 4 above)
Comment (PavelBal) [reply tests/testthat/datetime-functions.r:29]: Yes xD

**Comment by PavelBal** [comment]
> not that I would understand it in detail or be sure that it still works after that 

Neither do I :D I will change the link to distrib in a next PR

**Pull request merged**